flux-cli/src/test/java/com/marklogic/flux/api/ClassifierOptionsTest.java

```diff
@@ -13,15 +13,15 @@
 class ClassifierOptionsTest extends AbstractOptionsTest {
```

Comment:
Convert this test to use the CLI options instead of the API options. The CLI options are what will be used likely 98% of the time. The only reason `EmbedderOptionsTest` exists was to have one non-import test that sanity checks all the options. But most of the testing of embedder options is done via CLI tests using `--embedder-...`.

Reply:
I'm not sure what needs to be done for this. 
I considered mimicking `ImportAndSplitFilesTest` but that's actually doing the import. If I do that, I'll need to add the `@EnabledIfEnvironmentVariable` construct.
Is that what you want?

Reply:
Just use the CLI options instead of the API methods - e.g.:

```
ImportFilesCommand command = (ImportFilesCommand) getCommand(
            "import-files",
            "--connection-string", "user:password@host:8001",
            "--path", "anywhere",
            "--embedder", "azure",
            "-Ekey=value",
            "-EotherKey=otherValue",
            "--embedder-chunks-json-pointer", "/some/chunks",
            "--embedder-text-json-pointer", "/my-text",
            "--embedder-chunks-xpath", "/xml/chunks",
            "--embedder-text-xpath", "/xml/text",
            "--embedder-embedding-name", "stuff",
            "--embedder-embedding-namespace", "org:example",
            "--embedder-batch-size", "14"
        );
```